projects/aca-about/src/lib/aca-about.module.ts

```diff
@@ -48,4 +42,13 @@ export class AcaAboutModule {
       'app.about.component': AboutComponent
     });
   }
+
+  public static forRoot(environment: any): ModuleWithProviders<AcaAboutModule> {
+    return {
+      ngModule: AcaAboutModule,
+      providers: [
+        {provide: 'environment', useValue: environment}
```

Comment:
Although you can use strings as injection tokes, it is not recommended. I would suggest using InjectionToken.